### PR TITLE
Use Different namespace for debug for Baremetal Deployment

### PR DESCRIPTION
**Pull request by prsurve**
Fixes: #6439 

Signed-off-by: prsurve <prsurve@redhat.com>

### CODE REVIEW

File: ocs_ci/deployment/baremetal.py [hunk 1]
```diff
@@ -35,6 +35,7 @@
     TimeoutSampler,
     add_chrony_to_ocp_deployment,
 )
+from ..ocs.utils import label_pod_security_admission
```
Comment (ebenahar) [inline ocs_ci/deployment/baremetal.py:38]: I think it's problematic to have relative imports as the directories structure may change in the future

File: ocs_ci/deployment/baremetal.py [hunk 2]
```diff
@@ -684,11 +685,18 @@ def clean_disk():
     Perform disk cleanup
     """
     lvm_to_clean = []
+    bm_debug_ns = "baremetal-debug-ns"
```
Comment (ebenahar) [inline ocs_ci/deployment/baremetal.py:688]: please save in constants.py

File: ocs_ci/deployment/baremetal.py (hunk 1 above)
Comment (prsurve) [reply ocs_ci/deployment/baremetal.py]: Done

File: ocs_ci/deployment/baremetal.py (hunk 2 above)
Comment (prsurve) [reply ocs_ci/deployment/baremetal.py]: Done

**Review by vavuthu — APPROVED** [review]

File: ocs_ci/deployment/baremetal.py [hunk 3]
```diff
@@ -685,10 +686,16 @@ def clean_disk():
     """
     lvm_to_clean = []
     workers = get_nodes(node_type="worker")
+
     ocp_obj = ocp.OCP()
+    ocp_obj.new_project(project_name=constants.BM_DEBUG_NODE_NS)
```
Comment (dahorak) [inline ocs_ci/deployment/baremetal.py:691]: Where is the `constants.BM_DEBUG_NODE_NS` defined? (I don't see it in the [`constants.py`](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/constants.py) module and also it is not added in this PR.

**Review by dahorak — CHANGES_REQUESTED** [review]
If I didn't miss anything, there is missing definition of `constants.BM_DEBUG_NODE_NS`.

File: ocs_ci/deployment/baremetal.py (hunk 3 above)
Comment (prsurve) [reply ocs_ci/deployment/baremetal.py:691]: Updated

**Review by dahorak — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Pull request merged**